composer.json

```diff
@@ -14,6 +14,7 @@
         "symfony/dependency-injection": "^3.3||^4.3",
         "symfony/process": "^2.1||^4.1",
         "symfony/proxy-manager-bridge": "^3.3||^4.3",
+        "symfony/yaml": "^3.3||^4.0",
         "monolog/monolog": "^1.16",
         "magento/quality-patches": "^1.0.0"
```

Comment:
The packages have a dependency on each other and the composer resolves them without issues. Why it should be moved?

Reply:
Having circular dependency is never good idea, in some cases, it lead to neverending dependency resolving and other problems. This may be either included in ECE-Tools or MCC

Reply:
And circular dependency consieded as "bad smell" from Architecture Design standpoint

Reply:
> Having circular dependency is never good idea, in some cases, it lead to neverending dependency resolving and other problems. This may be either included in ECE-Tools or MCC

I agree with you, in case of recurrent read it can lead to neverending resolving.
In the current case we A => B and B => A. Composer resolves it well.
Otherway, if we move dependency on quality-patches to ece-tools, it will be impossible to install magento-cloud-patches from the composer separately.

Reply:
"magento/quality-patches" dependency was moved to suggest section
@shiftedreality please review